Buildings/Experimental/DHC/Loads/BaseClasses/Examples/BaseClasses/BuildingTimeSeries.mo

```diff
@@ -147,18 +151,20 @@ model BuildingTimeSeries
     offset={0,0,0},
     columns={2,3,4},
     smoothness=Modelica.Blocks.Types.Smoothness.MonotoneContinuousDerivative1)
-    "Reader for thermal loads (y[1] is cooling load, y[2] is heating load)"
+    "Reader for thermal loads (y[1] is cooling load, y[2] is space heating load, y[3] is sanitary water heat load)"
```

Comment:
I recommend calling "domestic hot water load" instead of "sanitary" to be consistent with typical convention.  This is also used elsewhere in model.